### PR TITLE
feat: websites where you can use alby

### PR DESCRIPTION
**Pull request by umarrg**

### CODE REVIEW

**Comment by bumi** [comment]
thanks a lot for the PR. 
I still need to review it in detail, but I am wondering if we can somehow make it simple. 
It's quite a lot of code for showing those boxes. Maybe we could simple list the boxes... so we have one component and one array we iterate over?

**Comment by bumi** [comment]
I wanted to follow up on this. do you think we can make this change a bit simpler/smaller?

**Comment by umarrg** [comment]
Ok

**Comment by bumi** [comment]
thanks for your work on this. I think it is great when we get this one in soon. 

But I still think we should simplify this and only show a nice list of websites without the need of adding so much new components and colors etc. 
just a "stipid" list. do you see a problem with that?

**Comment by umarrg** [comment]
Ok

**Comment by bumi** [comment]
@umarrg shall we finalize this? what do we need?

**Comment by umarrg** [comment]
Pls let me know if there's anything else

**Comment by escapedcat** [comment]
It's a quite a big PR and hard to comprehend for an outsider like me.

Would be super helpful if this PR had a description.
Like what has been changed and why. Even better with screenshots before/after.
Maybe there's a related issue? If yes would be great to link it.
This would help people to understand the intention of this PR.

Maybe these things have been discussed somewhere than it would be great to add that discussion into the description.

Thanks for you efforts! ❤️

**Review by escapedcat — APPROVED** [review]

**Comment by bumi** [comment]
Currently this loads quite some images from twitter and other sources. I think we should avoid external sources and upload those images to our storage. Then we can be sure that those images not simply change or disappear and also users don't access external resources by default (but only trusted alby resources) 

@MoritzKa can you upload those images to our DO storage. 
@kiwiidb I think the DO storage does not have a custom domain. we should setup something like `static.getalby.com` or `assets.getalby.com` for this. can you configure that? (or does this needs to be done directly as DNS records?)

**Review by bumi — CHANGES_REQUESTED** [review]
Currently this loads quite some images from twitter and other sources. I think we should avoid external sources and upload those images to our storage. Then we can be sure that those images not simply change or disappear and also users don't access external resources by default (but only trusted alby resources)

@MoritzKa can you upload those images to our DO storage.
@kiwiidb I think the DO storage does not have a custom domain. we should setup something like static.getalby.com or assets.getalby.com for this. can you configure that? (or does this needs to be done directly as DNS records?)

**Comment by MoritzKa** [comment]
I can do that. Is there any specific image size required or can I go with the size of the image that we use right now?
I just need a place to upload them.

**Comment by bumi** [comment]
> I can do that. Is there any specific image size required or can I go with the size of the image that we use right now?
I just need a place to upload them.

They should all have the same format and size with a reasonable resolution. also check the file-size that it is not more than a low 100s of KB, metadata should be removed for PNGs. but probably SVG versions are the best. 
@dylancom do you have suggestions for that, maybe. - I'd have to do some research.

You still have that DO storage that you used for makers.bolt.fun? let's upload it there to a new folder.

**Comment by bumi** [comment]
we should use the static.getalby.com domain instead of the digital ocean domain directly. 
@kiwiidb what do I need to do to set it up?

**Comment by bumi** [comment]
@MoritzKa what about?
* makers.bolt.fun
* geyser.fund

**Comment by MoritzKa** [comment]
Yes. I will add them once we figured out where to store the logos. 
Let's get it into the next release. We can always add more websites later on.

**Comment by bumi** [comment]
no, the URL must be an alby URL, before that I will not merge it. 
logos will be stored on DO as you uploaded them. 

just the URL will change.

File: src/app/screens/Publishers/index.tsx
Comment (bumi) [inline src/app/screens/Publishers/index.tsx:53]:
```suggestion
        Some websites where you have used Alby before
```

should we indicate here that it is only a selection?

File: src/app/screens/Publishers/index.tsx
Comment (bumi) [inline src/app/screens/Publishers/index.tsx:74]: is `h-32` or something maybe enough? I feel like the boxes are pretty big. 
Every description seems to fit except the lightning store one - but there we can shorten the description.

**Review by bumi — COMMENTED** [review]
@MoritzKa we now have `https://cdn.getalby-assets.com/` as domain.

File: src/app/screens/Publishers/index.tsx
Comment (dylancom) [reply src/app/screens/Publishers/index.tsx:53]: I'm not too fond of "Some websites".

**Pull request merged**
